kindly check the solution @tavisca-nmulani

tavisca-tgoyal opened the pull request.

tavisca-tgoyal committed:
Update Program.cs

tavisca-tgoyal committed:
Program login in human friendly format.

tavisca-nmulani reviewed (changes requested):
Please add validations as mentioned in constraints section.

tavisca-tgoyal committed:
added validations

added validations according to the constraints defined in the problem statement. There is a total of 4 constraints.
This is the first time I've performed validations. So, don't know what's the exact convention of doing that. 
This program will first validate the input. It doesn't go forward in execution if the input is failed according to our constraints.

tavisca-tgoyal commented:
> Please add validations as mentioned in constraints section.

@tavisca-nmulani done!
